.github/workflows/lint_test.yml

```diff
@@ -47,13 +47,5 @@ jobs:
       - name: Run chart-testing (lint)
         run: make lint
 
-      - name: Check Docs
```

Comment:
This will run on release PRs and push any changes instead of running on each PR and failing requiring another commit to update the docs